src/clj/ataru/hakija/hakija_application_service.clj

```diff
@@ -184,8 +184,12 @@
                                               :key
                                               application-store/get-application-hakukohde-reviews)
         form                          (cond (some? (:haku application))
-                                            (hakija-form-service/fetch-form-by-haku-oid-and-id-cached
-                                             form-by-haku-oid-and-id-cache
+                                            (hakija-form-service/fetch-form-by-haku-oid-and-id
+                                             form-by-id-cache
+                                             tarjonta-service
+                                             koodisto-cache
+                                             organization-service
+                                             ohjausparametrit-service
                                              (:haku application)
                                              (:form application)
                                              (util/application-in-processing? application-hakukohde-reviews)
```

Comment:
Nämä signaturet alkavat olla aika pitkiä. Jos dependency injection ei sovi, pitäisikö miettiä, että olisi vaikka jokin `caches`-mäppi, jota passailtaisi eteenpäin? Toki silloin hämärtyy se, missä kaikissa paikoissa tarvitaan juuri tiettyjä cacheja (tai muita serviceitä, jos tekisi jonkun vielä yleisemmän `services`:in tms).